src/odemis/driver/zeiss.py

```diff
@@ -297,6 +301,20 @@ def GetStagePosition(self):
         vals = s.split(b' ')
         return tuple(float(i) for i in vals[0:3]) + (vals[3] == b"1.0",)
 
+    def GetStagePosition6(self):
+        """
+        Read the absolute position of the stage for all 6 axes.
+        Non-existing axes get a float value of 0.0 (cannot be left out).
+        :return: float, float, float, float, float, float, bool: x, y, z in mm and t, r in deg and m in mm,
+        stage moving as bool.
+        """
+        # Return something like:
+        # 65.60162 64.75846 38.91104 0.23 259.22 0.0 0.0
+        # stage moving is either 0.0 or 1.0
+        s = self._SendCmd(b'C95?')
+        vals = s.split(b' ')
```

Comment:
```suggestion
        vals = s.strip().split(b' ')
```
Maybe better?

Reply:
I think it is fine for now, since we already used the command GetStagePosition for a while and the output from this is reliable. The GetStagePosition6 is derived from this command.